doc/usage/bfcli.rst

```diff
@@ -101,9 +101,14 @@ Rules are defined such as:
 With:
   - ``$MATCHER``: zero or more matchers. Matchers are defined later.
   - ``counter``: optional literal. If set, the filter will counter the number of packets and bytes matched by the rule.
-  - ``$VERDICT``: action taken by the rule if the packet is matched against **all** the criteria: either ``ACCEPT`` or ``DROP``.
+  - ``$VERDICT``: action taken by the rule if the packet is matched against **all** the criteria: either ``ACCEPT``, ``DROP`` or ``CONTINUE``.
+    - ``ACCEPT``: Forward the packet to the kernel
+    - ``DROP``: Discard the packet.
+    - ``CONTINUE``: Continue processing subsequent rules.
 
-In a chain, as soon as a rule matches a packet, its verdict is applied, and the subsequent rules are not processed. Hence, the rules' order matters. If no rule matches the packet, the chain's policy is applied.
+In a chain, as soon as a rule matches a packet, its verdict is applied. If the verdict is ``ACCEPT`` or ``DROP``, the subsequent rules are not processed. Hence, the rules' order matters. If no rule matches the packet, the chain's policy is applied.
+
+Note ``CONTINUE`` means a packet can be counted more than once if multiple matchers specify ``CONTINUE`` and ``counter``.
```

Comment:
"if multiple matchers" -> "if multiple rules"

For reference, a rule contains 1+ matchers. The rule defines the verdict if all the matchers matches.